package.json

```diff
@@ -16,7 +16,13 @@
     "build-js": "yarn build-js:application && yarn build-js:all",
     "build-js:application": "mkdir -p dist/public/javascripts; uglifyjs src/assets/javascripts/application.js --beautify -o dist/public/javascripts/application.js",
     "build-js:all": "mkdir -p dist/public/javascripts; uglifyjs node_modules/govuk-frontend/govuk/all.js node_modules/hmpo-components/all.js --beautify -o dist/public/javascripts/all.js",
+    "lint": "eslint .",
+    "prettier": "prettier \"**/*.{js,md}\" --check",
+    "prettier:fix": "yarn prettier --write",
+    "format": "yarn prettier:fix && yarn lint",
```

Comment:
Wondering if I should keep the name convention of "lint" and expand the "lint" script to include "prettier --fix"? Since we use "lint" in the other repos